src/lib/widgets/user/ui/avatar.svelte

```diff
@@ -0,0 +1,7 @@
+<script lang="ts">
+  export let image: string;
+</script>
+
+<div class="aspect-square h-12 w-12 rounded-full overflow-hidden">
+  <img src={image} alt="user avatar" />
```

Comment:
issue (non-blocking): I'm not sure if this alt text is actually helpful. Alt text works best when it describes the content of an image, not its purpose, which is usually obvious from context